readeckbot/main.py

```diff
@@ -219,7 +219,7 @@ async def register_and_fetch_token(update: Update, username: str, password: str)
     Then obtain the token via the API.
     """
     command = (
-        ["readeck", "user"] + (["-config", READECK_CONFIG] if READECK_CONFIG else []) + ["-u", username, "-p", password]
+        ["./readeck", "user"] + (["-config", READECK_CONFIG] if READECK_CONFIG else []) + ["-u", username, "-p", password]
```

Comment:
nop. 

 the thing is readeck should be installed in somewhere in the `$PATH` (like `/user/local/bin/`)  , but then executed within the root of readeckbot. 

merging this with `["./readeck` ... would break production for instance.   Just clarify where to put `readeck` in the readme, please.